packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,143 @@
+import {
+  kubernetesCurrentContextConfigMaps,
+  kubernetesCurrentContextDeployments,
+  kubernetesCurrentContextIngresses,
+  kubernetesCurrentContextNodes,
+  kubernetesCurrentContextPersistentVolumeClaims,
+  kubernetesCurrentContextRoutes,
+  kubernetesCurrentContextSecrets,
+  kubernetesCurrentContextServices,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+import { NO_CURRENT_CONTEXT_ERROR } from '/@api/kubernetes-contexts-states';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
+import KubeApplyYamlButton from './KubeApplyYAMLButton.svelte';
+import KubernetesDashboardGuideCard from './KubernetesDashboardGuideCard.svelte';
+import KubernetesDashboardResourceCard from './KubernetesDashboardResourceCard.svelte';
+import KubernetesEmptyPage from './KubernetesEmptyPage.svelte';
+
+let welcome = $derived($kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR);
+let currentContextName = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
+
+let nodeCount = $state<number>(0);
+let deploymentCount = $state<number>(0);
+let serviceCount = $state<number>(0);
+let ingressCount = $state<number>(0);
+let routeCount = $state<number>(0);
+let pvcCount = $state<number>(0);
+let configMapCount = $state<number>(0);
+let secretCount = $state<number>(0);
+
+let unsubscribers: Unsubscriber[] = [];
+
+onMount(async () => {
+  unsubscribers.push(
+    kubernetesCurrentContextNodes.subscribe(nodes => {
+      nodeCount = nodes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextDeployments.subscribe(deployments => {
+      deploymentCount = deployments.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextServices.subscribe(services => {
+      serviceCount = services.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextIngresses.subscribe(ingresses => {
+      ingressCount = ingresses.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextRoutes.subscribe(routes => {
+      routeCount = routes.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextPersistentVolumeClaims.subscribe(pvcs => {
+      pvcCount = pvcs.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextConfigMaps.subscribe(configMaps => {
+      configMapCount = configMaps.length;
+    }),
+  );
+  unsubscribers.push(
+    kubernetesCurrentContextSecrets.subscribe(secrets => {
+      secretCount = secrets.length;
+    }),
+  );
+});
+
+onDestroy(async () => {
+  unsubscribers.forEach(unsubscriber => unsubscriber());
+});
```

Comment:
suggestion: using derived you'll be able to remove it

Reply:
Fixed.